src/specs/models/processing-config-bodies/ProcessingConfigMeta.v1.yaml

```diff
@@ -0,0 +1,13 @@
+title: Metadata
+type: object
+description: Metadata regarding the progress of the data processing steps.
+properties:
+  complete:
+    type: boolean
+  stepsDone:
+    type: array
+    items:
+      type: string
+required:
+  - complete
+  - stepsDone
```

Comment:
New spec for metadata such as completion and steps done.

Reply:
@aerlaut can you review this?

Reply:
Yup, looks good. It stores data on which steps are completed, and whether all the steps are completed.